hoomd/hpmc/ShapeSphere.h

```diff
@@ -36,10 +29,19 @@
 namespace hpmc
 {
 
-//! Base class for parameter structure data types
-struct param_base
+/** HPMC shape parameter base class
+
+    HPMC shape parameters must be aligned on 32-byte boundaries for AVX acceleration. The ShapeParam
+    base class implements the necessary aligned memory allocation operations. It also provides
+    empty load_shared and allocated_shared implementations which enabled caching deep copied managed
+    data arrays in shared memory.
+
+    \todo Move base methods out into their own file. ShapeSphere.h will then no longer need to be
```

Comment:
Just for easier grepping since we don't build C++ documentation any more.
```suggestion
    TODO Move base methods out into their own file. ShapeSphere.h will then no longer need to be
```